src/list/list.service.ts

```diff
@@ -0,0 +1,50 @@
+import { v4 as uuidv4 } from 'uuid';
+import { addedData201, internalServerError500, OK200 } from '../constants';
+import { getListOfContacts } from '../contact/contact.utils';
+import { ContactsList } from '../contactsList/contactsList.model';
+import { WorkWithDB } from '../interfaces';
+import { showData } from '../utils/utils';
+import { List } from './list.model';
+
+export const createNewList: WorkWithDB = async (req) => {
+  const { name } = JSON.parse(req.body.list) as { name: string };
+  try {
+    const res = await List.create({
+      uuid: uuidv4(),
```

Comment:
This value should be auto generate by Model

Reply:
Changed